src/utils/_Popper.js

```diff
@@ -94,6 +95,11 @@ class Popper extends React.Component {
                     if (!show) {
                         return null;
                     }
+                    const currentParentElement = parentElement && parentElement.current;
```

Comment:
Regarding my other comment, `parentRef` would be _slightly_ clearer here that we could access a property `.current`.